hack/jenkins/build_iso.sh

```diff
@@ -40,12 +40,16 @@ else
 	export ISO_BUCKET
 fi
 
-make release-iso
+make release-iso | tee iso-logs.txt
 # Abort with error message if above command failed
 ec=$?
 if [ $ec -gt 0 ]; then
 	if [ "$release" = false ]; then
-		gh pr comment ${ghprbPullId} --body "Hi ${ghprbPullAuthorLoginMention}, building a new ISO failed, please try again."
+		err=$(tail -50 iso-logs.txt)
```

Comment:
50 lines would probaby not give much info lets do 1000
would it be possible to upload the log and provide link to the file ?

Reply:
the logs are already uploaded, the tee is only specifically for this purpose.